core/src/main/java/org/elasticsearch/common/io/stream/StreamableReader.java

```diff
@@ -20,11 +20,17 @@
 
 import java.io.IOException;
 
+/**
+ * Implementers can be read from {@linkplain StreamInput} by calling their {@link #readFrom(StreamInput)} method.
+ *
+ * It is common for implementers of this interface to declare a <code>public static final</code> instance of themselves named PROTOTYPE so
+ * users can call {@linkplain #readFrom(StreamInput)} on it. It is also fairly typical for readFrom to be implemented a method that just
```

Comment:
I think "to be implemented a method" needs to be changed to "to be implemented **as** a method"?